src/main/resources/pom-servers.xml

```diff
@@ -151,7 +151,7 @@
                             </execution>
                         </executions>
                         <configuration>   
-                            <appArchive>${project.build.directory}/${final.name}.war</appArchive>
+                            <archive>${project.build.directory}/${final.name}.war</appArchive>
```

Comment:
Closing tag is not the same as opening tag.

Reply:
I suggest to delete this line altogether as it is not needed.